policies/travel-policy.md

```diff
@@ -15,9 +15,9 @@ approved travel expenses.
 Scope
 -----
 
-The Policy applies to all Committee Members, all persons under contract
-at OpenSSL, as well as other persons as designated by the OMC from time
-to time (subsequently known collectively as *Traveller*).
+The Policy can be applicable to all Committee Members, all persons under
```

Comment:
Nit: I'm reading "can be applicable" as an uncertainty ("it can be applicable, but might not")... is that the tone we want to set in a policy like this?  If not, would "is applicable" set a more certain tone?

Reply:
Updated to "is applicable"